dataprep/data_connector/schema.json

```diff
@@ -61,6 +61,19 @@
                 "params": {
                     "$ref": "#/definitions/fields"
                 },
+                "pagination": {
+                    "$id": "#/properties/request/properties/pagination",
+                    "type":"object",
+                    "additionalProperties": {
```

Comment:
Can you make the keys of pagination deterministic and forbid additional properties?